tests/components/google_generative_ai_conversation/test_conversation.py

```diff
@@ -34,6 +44,15 @@ async def test_default_prompt(
     if agent_id is None:
         agent_id = mock_config_entry.entry_id
 
+    if allow_hass_access:
```

Comment:
Make a parameter for the data to update the options instead. Empty dict or `{CONF_LLM_HASS_API: llm.LLM_API_ASSIST}`.